public/js/pimcore/document/editables/date.js

```diff
@@ -29,12 +29,14 @@ pimcore.document.editables.date = Class.create(pimcore.document.editable, {
 
     render: function () {
         this.setupWrapper();
+        
+        if (this.config.format && this.config.format.includes('%')) {
```

Comment:
@kingjia90 maybe we should also update docs to defined which formats are supported by Extjs? would be nice with a link

Reply:
Providing a PR for that https://github.com/pimcore/pimcore/pull/16275